OpenEdXMobile/res/drawable/search_icon.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
This file doesn't look properly formatted as the original one in the support package.

As a convention, we should copy the License from the file as well and keep the file's name too i.e. `abc_ic_search_api_material.xml` to `ic_search_api_material.xml` (just remove the abc prefix from start).

Reply:
If we don't want to copy the License as is, we can just add a comment as to why we copied this resource instead of using the one from the support lib.